source/renderer/app/analytics/MatomoClient.ts

```diff
@@ -40,18 +39,23 @@ export class MatomoClient implements AnalyticsClient {
     });
   };
 
-  sendEvent = async (category: string, action: string): Promise<void> => {
+  sendEvent = async (
+    category: string,
+    action: string,
+    name?: string
+  ): Promise<void> => {
```

Comment:
In general I'd agree, however in this case the third parameter is optional and is not used in 90% cases. I think it would make sense if we have more than 3 parameters or various combinations of parameters.